README.md

```diff
@@ -141,14 +143,18 @@ Usage of amazon-eks-pod-identity-webhook:
 When the `aws-default-region` flag is set this webhook will inject `AWS_DEFAULT_REGION` and `AWS_REGION` in mutated containers if `AWS_DEFAULT_REGION` and `AWS_REGION` are not already set.
 
 
+## Container Images
+
+Container images for amazon-eks-pod-identity-webhook can be found on [Docker Hub](https://hub.docker.com/r/amazon/amazon-eks-pod-identity-webhook).
+
 ## Installation
 
 ### In-cluster
 
 You can use the provided configuration files in the `deploy` directory, along with the provided `Makefile`
 
 ```
-make cluster-up IMAGE=602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pod-identity-webhook:latest
+make cluster-up IMAGE=amazon/amazon-eks-pod-identity-webhook:<tag>
```

Comment:
Can we replace with a tag that we have tested and know it's working?

Reply:
@anguslees reported some issue pulling the only existing tag (`2db5e53`). I re-ran our github action and it's now working. So I'll put `2db5e53` in the README for now.